### ISSUE TITLE
Consider using a trigger for timestamps

### ISSUE THREAD

**Issue by paulcsmith** [luckyframework/avram#337]
https://x-team.com/blog/automatic-timestamps-with-postgresql/

**Comment by bdtomlin** [comment]
I like it! One of the great things about being committed to PostgreSQL is the ability to do stuff like this.

**Comment by paulcsmith** [comment]
Agreed!! This is really why I don't think Avram will ever support other databases. Too many goodies in PG and I don't want to be slowed down with figuring out multiple ways to do everything.

I'd like to document how to use other ORMs if you *really* need mysql or something (legacy databases, corp reqs, etc.) but for now though just PG seems to be a good way to go!
